codefresh/pipeline-creator/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -x
```

Comment:
Don't use `set -x` when commands will be printed with secrets. 

Do use `set -e` to exit on all errors.